snmp.yml

```diff
@@ -1953,6 +1955,7 @@ cisco_wlc:
   - 1.3.6.1.4.1.14179.2.2.2.1.2
   - 1.3.6.1.4.1.14179.2.2.2.1.4
   - 1.3.6.1.4.1.14179.2.2.6.1
+  get: []
```

Comment:
Hmm, are we missing an omitempty?

Reply:
Yes indeed, added for both Get and Walk, and cleaned up snmp.yml.